config.json

```diff
@@ -961,6 +961,18 @@
         "loops"
       ]
     },
+    {
+      "slug": "zebra-puzzle",
```

Comment:
If this has difficulty 6 then it should probably be placed before two-bucket? 🙂 But I think it might be more difficult than a 6? Especially since the exercise format offers very little guidance so they can't do it as "step by step" as most other exercises. So maybe it should be a 7 or 8? What do you think @sjwarner-bp @lemoncurry? 🙂